### ISSUE TITLE
HttpClient PostAsync while using Web Proxy

### ISSUE THREAD

**Issue by jpreese** [justcoding121/Titanium-Web-Proxy#59]
I'm using Titanium Web Proxy to be able to monitor the traffic on my network, with the end goal of being able to get information about each URL visited, and potentially block them.

I've got it to the point where I can see the incoming URLs, but everytime I try and run HttpClient.PostAsync from within my application, I'm never able to get a response.  If I turn the proxy off, it works just fine.

**Comment by jpreese** [comment]
I just realized this question was asked previously, but it doesn't look like there was a resolution. Issue is #48.

The code to run PostAsync is as follows:

```
            using (var client = new HttpClient())
            {
                var values = new Dictionary<string, string>
                {
                    { "url", url }
                };

                var content = new FormUrlEncodedContent(values);
                var request = await client.PostAsync("http://sitereview.bluecoat.com/rest/categorization", content);
                var responseString = await request.Content.ReadAsStringAsync();

                var category = CreateSiteInfoJson(responseString);

                return category;
            }
```

It hands indefinitely and eventually cancels the task on await client.PostAsync(...)

**Comment by justcoding121** [comment]
Thanks for reporting this bug. I am looking at it.
Meanwhile you should be able to post successfully if you disable ExpectContinue in HttpClient like below.

`client.DefaultRequestHeaders.ExpectContinue = false;`

**Comment by jpreese** [comment]
Thanks!  Your ExpectContinue = false looks like it makes everything run smoothly.

**Comment by justcoding121** [comment]
Fixed in #64 on beta
